modules/ytp-assets-common/src/less/components/list--resource.less

```diff
@@ -24,18 +28,45 @@
 
     .resource-item__actions {
       text-transform: uppercase;
-      line-height: 35px;
+      line-height: 14px;
       font-weight: 600;
       margin-left: auto;
     }
 
+    .resource-item__info__title {
+      color: @opendata-brand-primary;
+      transition: color @transition-duration--short @transition-timing-function--default;
+      font-size: 16px;
+      font-weight: normal;
+      line-height: 24px;
+      font-family: @font-family-sans-serif;
+    }
+
     a {
       color: @opendata-brand-primary;
       transition: color @transition-duration--short @transition-timing-function--default;
+      font-size: 16px;
+      font-weight: 600;
+      line-height: 24px;
+      font-family: @font-family-sans-serif-semibold;
 
       &:hover {
         color: @opendata-brand-primary--highlight;
       }
     }
   }
 }
+
+.resource-list-apiset {
+  .resource-item {
+    padding: 11px 10px 11px 15px !important;
```

Comment:
Should we use variable `@padding__resource-list-item` in here, and then update that to match with this definition?